src/zac/reports/export.py

```diff
@@ -0,0 +1,101 @@
+from typing import Dict, Iterator, List
+
+import tablib
+from zgw_consumers.api_models.base import factory
+from zgw_consumers.api_models.catalogi import ZaakType
+from zgw_consumers.api_models.zaken import Status, Zaak, ZaakEigenschap
+from zgw_consumers.concurrent import parallel
+
+from zac.core.services import (
+    _get_from_catalogus,
+    get_status,
+    get_zaak,
+    get_zaak_eigenschappen,
+)
+from zac.elasticsearch.searches import search
+
+from .models import Report
+
+
+def _get_zaaktypen(report: Report) -> List[ZaakType]:
+    identificaties = report.zaaktypen
+
+    def _get_zaaktype_versions(identificatie: str):
+        return _get_from_catalogus(
+            "zaaktype", catalogus="", identificatie=identificatie
+        )
+
+    with parallel() as executor:
+        zaaktypen = executor.map(_get_zaaktype_versions, identificaties)
```

Comment:
why not something like:

```suggestion
    with parallel() as executor:
        zaaktypen = executor.map(
            lambda identificatie: _get_from_catalogus(
                "zaaktype", catalogus="", identificatie=identificatie
            ),
            identificaties,
        )
```
or does it mess with readability too much? :-)